src/startup.c

```diff
@@ -59,7 +59,7 @@ void dump_boot_args(struct boot_args *ba)
     printf("    stride:     0x%lx\n", ba->video.stride);
     printf("    width:      %lu\n", ba->video.width);
     printf("    height:     %lu\n", ba->video.height);
-    printf("    depth:      0x%lx\n", ba->video.depth);
+    printf("    depth:      %lubpp\n", ba->video.depth);
```

Comment:
Is it intentional that this now prints "65566bpp" for my MacBook with the retina flag set? That made me think I had a broken m1n1 version, and so I wasted some time trying to track it down...

Reply:
Should be fixed in 19b16f77d.

Reply:
Thanks, can confirm that fixes the issue!